### ISSUE TITLE
Undo migrations with down.sql?

### ISSUE THREAD

**Issue by josvazg** [uptrace/bun#750]
Could not find in the docs whether or now using `.down.sql` scripts is supported to go back to an earlier db state.

Could you confirm whether or not that is the case?

**Comment by vmihailenco** [comment]
Yes, you can check [this](https://github.com/uptrace/bun/tree/master/example/migrate/migrations) example which has `.down.sql` migrations.
